health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateQuestionContext.js

```
@@ -46,11 +50,13 @@ const CreateQuestionContext = ({ onSelect, ...props }) => {
             title: null,
             type: { "code": "SingleValueList" },
             value: null,
-            isRequired: false,
+            isRequired: true,
+            isActive: true
           },
         ];
         break;
```

Biome

[error] This code is unreachable



(lint/correctness/noUnreachable)